src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -436,6 +475,9 @@ def get_request_status(
         for item in paginated.items:
             item.identity = item.get_cached_identity_data()
 
+    for item in paginated.items:
+        attach_resume_instructions(item)
```

Comment:
Looping through all the privacy request items again, but this request is paginated, so it shouldn't be too many requests.